Cython/Compiler/Options.py

```diff
@@ -111,7 +111,7 @@ def __repr__(self):
 #: from their Python 2 names to their Python 3 names by Cython
 #: when building in Python 3.x,
 #: so that they do not get in the way even if this option is enabled.
-cache_builtins = True
+cache_builtins = False
```

Comment:
?

Reply:
Sorry, please just ignore that. I won't, ofc, ever merge this change. I just need to migrate the cache logic.